### ISSUE TITLE
Incorrect apply command behaviour

### ISSUE THREAD

**Issue by ChaserVasya** [slang-i18n/slang#224]
**Describe the bug**
`apply` command has incorrect behaviour.
**To Reproduce**
add new translation to base locale file
```dart
enum DateRange {
  day,
  week,
  month,
}
```
```json
  "dateRange(context=DateRange, param=type)" : {
    "day": "День",
    "week": "Неделя",
    "month": "Месяц"
  },
```
`dart run slang analyze`
Fill other translations from `_missing_translations.json`
`dart run slang apply`

output:
```dart
  "dateRange(context=DateRange, param=type)": {
    "day": "Day",
    "week": "Week",
    "month": "Month"
  },
  "dateRange": {
    "day": "Day",
    "week": "Week",
    "month": "Month"
  }
```
**Expected behavior**
output:
```dart
  "dateRange(context=DateRange, param=type)": {
    "day": "Day",
    "week": "Week",
    "month": "Month"
  },
```

**Comment by Tienisto** [comment]
Do you use the latest version?

**Comment by ChaserVasya** [comment]
@Tienisto 
3.29.0

**Comment by Tienisto** [comment]
This should be already fixed in 3.30.1
